ui/apps/platform/src/Containers/Clusters/AddClusterPrompt.tsx

```diff
@@ -11,8 +11,6 @@ import {
     TextVariants,
 } from '@patternfly/react-core';
 
-import SecurityFullColorPink from 'images/Security_FullColor_Pink.svg';
-
```

Comment:
Can we also `git rm ui/images/Security_FullColor_Pink.svg`,
if this is the only place that image is used?